### PR TITLE
refactor(components): component folder refactor

### PR DESCRIPTION
**Pull request by ydalir**

### CODE REVIEW

File: src/components/AccountModals/DeleteTavleModal/DeleteTavleModal.tsx [hunk 1]
```diff
@@ -7,7 +7,7 @@ import { Heading3, Paragraph } from '@entur/typography'
 import { GridContainer, GridItem } from '@entur/grid'
 import { PrimaryButton, SecondaryButton } from '@entur/button'
 import { useToast } from '@entur/alert'
-import { CloseButton } from '../../CloseButton/CloseButton'
+import { CloseButton } from '../../CloseButton'
```
Comment (lindtvedtsebastian) [inline src/components/AccountModals/DeleteTavleModal/DeleteTavleModal.tsx:10]: /components/CloseButton

File: src/components/AccountModals/LockModal/LockModal.tsx [hunk 2]
```diff
@@ -7,7 +7,7 @@ import { Heading3, Paragraph } from '@entur/typography'
 import { Modal } from '@entur/modal'
 import { GridContainer, GridItem } from '@entur/grid'
 import { PrimaryButton } from '@entur/button'
-import { CloseButton } from '../../CloseButton/CloseButton'
+import { CloseButton } from '../../CloseButton'
```
Comment (lindtvedtsebastian) [inline src/components/AccountModals/LockModal/LockModal.tsx:10]: samme her

File: src/components/AccountModals/LoginModal/EmailLogin/EmailLogin.tsx [hunk 3]
```diff
@@ -12,7 +12,7 @@ import { BackArrowIcon, ClosedLockIcon, EmailIcon } from '@entur/icons'
 import { PrimaryButton } from '@entur/button'
 import { Heading3, Link } from '@entur/typography'
 import { SmallExpandableAlertBox } from '@entur/alert'
-import { CloseButton } from '../../../CloseButton/CloseButton'
+import { CloseButton } from '../../../CloseButton'
```
Comment (lindtvedtsebastian) [inline src/components/AccountModals/LoginModal/EmailLogin/EmailLogin.tsx:15]: Generelt for alle closebutton

File: src/scenarios/Map/components/StopPlaceMarker/index.tsx [hunk 4]
```diff
@@ -4,9 +4,9 @@ import classNames from 'classnames'
 import { uniq } from 'lodash'
 import { useStopPlaceWithEstimatedCalls } from 'hooks/use-stop-place-with-estimated-calls/useStopPlaceWithEstimatedCalls'
 import { getIconColor } from 'utils/icon'
+import { TransportModeIcon } from 'assets/icons/TransportModeIcon'
+import { WalkTrip } from 'components/WalkTrip/WalkTrip'
```
Comment (lindtvedtsebastian) [inline src/scenarios/Map/components/StopPlaceMarker/index.tsx:8]: index

Comment (BirgitteBS) [reply src/scenarios/Map/components/StopPlaceMarker/index.tsx:8]: lindex

File: src/scenarios/DashboardHeader/index.tsx [hunk 5]
```diff
@@ -3,8 +3,8 @@ import { Link } from 'react-router-dom'
 import classNames from 'classnames'
 import { useSettings } from 'settings/SettingsProvider'
 import { TavlaLogo } from 'assets/icons/TavlaLogo'
+import { Clock } from 'components/Clock/Clock'
```
Comment (BirgitteBS) [inline src/scenarios/DashboardHeader/index.tsx]: index

**Review by BirgitteBS — APPROVED** [review]

**Pull request merged**